scripts/import-npa/natural_protected_areas.sh

```
@@ -0,0 +1,461 @@
+  printf "%s\n" "${shapefiles[@]}" > "$shapefile_list"
+  
+  echo "$shapefile_list"
+}
+
+# Convert shapefiles to SQL and combine them
+convert_shapefiles_to_sql() {
+  local polygons_list=$1  # Expecting the polygons.txt file
+  
+  local schema_polygons="${TEMP_DIR}/schema_polygons.sql"
+  local polygons="${TEMP_DIR}/polygons.sql"
+  > "$schema_polygons"
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)